docs/src/components/tabs/index.css

```
@@ -0,0 +1,55 @@
+  font-family: var(--font-commit-mono);
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.tabs .list .tab .indicator {
+  z-index: 1;
+  position: absolute;
+  inset: 0;
+  border: 0.0625rem solid var(--neutral-04);
+  border-radius: var(--rounded-sm);
+  corner-shape: squircle;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)